src/components/tl/ucc_tl.h

```diff
@@ -104,7 +104,7 @@ typedef struct ucc_team_multiple_req {
         ucc_tl_team_t           *team;
         ucc_base_team_params_t   param;
         ucc_status_t             status;
-    } descs[1];
```

Comment:
this is wrong change - plz remove. Take a look at ucc_team_multiple_req_alloc function, the proper size is used there. descs[1] is used for convenience of reference

Reply:
removed